manifest.yml

```diff
@@ -0,0 +1,8 @@
+---
+path: ./docs
+applications:
+- name: fec-pattern-library
+  memory: 64M
+  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
```

Comment:
Oops, @jmcarp is right, I just forgot how to specify the system buildpack. I'll update it.